toolkit/scripts/imggen.mk

```
@@ -16,6 +16,10 @@ initrd_config_json       = $(RESOURCES_DIR)/imageconfigs/iso_initrd_arm64.json
 else
 initrd_config_json       = $(RESOURCES_DIR)/imageconfigs/iso_initrd.json
 endif
+initrd_assets_dir        = $(RESOURCES_DIR)/imageconfigs/additionalfiles/iso_initrd/
+initrd_scripts_dir       = $(RESOURCES_DIR)/imageconfigs/postinstallscripts/iso_initrd/
+initrd_packages_dir      = $(RESOURCES_DIR)/imageconfigs/packagelists
+initrd_assets_files      = $(shell find $(initrd_assets_dir) $(initrd_scripts_dir) $(initrd_packages_dir))
```

Could we introduce this small change to be more precise about the dependencies?

```suggestion
initrd_packages_json     = $(RESOURCES_DIR)/imageconfigs/packagelists/iso-initrd-packages.json
initrd_assets_files      = $(initrd_packages_json) $(shell find $(initrd_assets_dir) $(initrd_scripts_dir))
```

---

Sure, I'll need to track all three files though

---

https://github.com/microsoft/CBL-Mariner/pull/4601/commits/e08c6f87961c902bea41320000ad2f2079e3b9c1